package.json

```diff
@@ -31,6 +31,7 @@
     "@formatjs/intl-segmenter": "^11.7.3",
     "@livekit/components-core": "^0.12.0",
     "@livekit/components-react": "^2.0.0",
+    "@livekit/protocol": "^1.33.0",
```

Comment:
Would be nice to have a second set of eyes confirming if its okay to add this explicitly.

Reply:
Yep, if we use it, we depend on it